spec/spec_helper_acceptance.rb

```diff
@@ -1,16 +1,11 @@
+# frozen_string_literal: true
+
 require 'beaker-rspec'
 require 'tmpdir'
 require 'yaml'
 require 'simp/beaker_helpers'
 include Simp::BeakerHelpers
 
```

Comment:
There is still a windows acceptance test suite and it is still being executed in GitLab.  Isn't the Windows configuration required for that test.

Reply:
That test successfully ran a month ago.

Reply:
@op-ct  I did a diff between this and the last passing Windows tests and I don't see anything that would have caused an issue. Could you take a look?